.github/workflows/next_build.yml

```diff
@@ -1,46 +1,15 @@
-name: Master build
+name: Next build
 
 on:
   push:
     branches:
       - next
 
 jobs:
-
-  release:
-    runs-on: ubuntu-20.04
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: node.js 16
-        uses: actions/setup-node@v2
-        with:
-          node-version: 16
-          registry-url: https://registry.npmjs.org/
-          cache: 'npm'
-
-      - name: set up git
-        run: git config --global url."https://github.com/".insteadOf "ssh://git@github.com/"
-
-      - name: installing
-        run: npm ci
-
-      - name: setup environment
-        run: source ./node_modules/.bin/setup-environment $PWD website-dao-dev
-
-      # TODO: enabled linters
-      # - name: linters
-      #   uses: wearerequired/lint-action@v1
-      #   with:
-      #     prettier: true
-      #     prettier_extensions: ts,tsx,js,jsx,,json,md
-      #     continue_on_error: false
-
-      - name: testing
-        run: npm test
-        env:
-          DISCOURSE_API_KEY: ${{ secrets.DISCOURSE_API_KEY }}
-
-      - name: building
-        run: npm run build
+  build:
+    uses: decentraland/actions/.github/workflows/build-quay-main.yml@main
+    with:
+      service-name: governance
+    secrets:
+      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+      QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
```

Comment:
I think you are forgetting to deploy the image to a specific env https://github.com/decentraland/sdk-content-server/blob/main/.github/workflows/docker.yml#L13

Reply:
Done! So if we want to deploy to all 3 environments we need to create a new workflow for each one. Right?

Reply:
separate them by space `dev stg prd`